### PR TITLE
Password Complexity Indicator doc

### PR DESCRIPTION
**Pull request by Sinan997**
doc

### CODE REVIEW

File: docs/en/UI/Angular/Password-Complexity-Indicator.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+`PasswordComplexityIndicatorService` is for Calculating Password Complexity.
+
+# Service
+### password-complextiy-indicator.service.ts
+```ts
+import { Injectable } from "@angular/core";
+import { ProgressBarStats } from '../models/password-complexity';
+
+@Injectable({providedIn: 'root'})
+  colors: string[] = [
```
Comment (mahmut-gundogdu) [inline docs/en/UI/Angular/Password-Complexity-Indicator.md]: where os the class name ? the code is broken

File: docs/en/UI/Angular/Password-Complexity-Indicator.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      if(isValid){
+        passedCounter++;
+      }
+    })
+    return { bgColors:this.colors, passedTestCount: passedCounter};
+  }
+```
+- As you can see from the code above we set default regular expressions requirements and colors. 
+- You can change tests and colors as you wish
+- **but remember if you change the requirements, it's length must be equal with the colors array**
```
Comment (mahmut-gundogdu) [inline docs/en/UI/Angular/Password-Complexity-Indicator.md]: Document must be simplified

**Review by braim23 — APPROVED** [review]

**Review by mahmut-gundogdu — CHANGES_REQUESTED** [review]
Move the docs 
https://github.com/abpio/abp-commercial-docs

**Comment by masum-ulu** [comment]
I'm closing this PR because the document must be in commercial repository. @Sinan997 will move the document

**Pull request closed without merging**